common/src/main/java/org/jboss/hal/testsuite/finder/PreviewFragment.java

```diff
@@ -45,10 +45,36 @@ public String getIncludedFromProfile() {
         return element.getText().substring(22).trim();
     }
 
-    public List<String> getDisplayedUnorderedList() {
-        By selector = ByJQuery.selector("ul li");
-        Graphene.waitAjax().until().element(root, selector).is().visible();
+    /**
+     * Extract unordered html-list from page.
+     *
+     * @param waitUntilValue  method will finishes successfully only when specified value occurs in list.
+     *                        It waits for global timeout at worst.
```

Comment:
I suggest to change `for global timeout` to `for global waitModelInterval timeout` since we have more predefined timeouts.